packages/math-input/src/components/__tests__/integration.test.tsx

```diff
@@ -80,7 +80,7 @@ function ConnectedMathInput({keypadConfiguration = defaultConfiguration}) {
 }
 
 describe("math input integration", () => {
-    let userEvent;
+    let userEvent: ReturnType<typeof userEventLib.setup>;
```

Comment:
I'm wondering if it'd be useful to alias this so it's clearer.

```
type UserEvent = ReturnType<typeof userEventLib.setup>;
```
Then
```
let userEvent: UserEvent;
```

Not a blocker, I'll defer to you.

Reply:
Nice.  The library even provides an exported type for this: `UserEvent`!